src/validation.cpp

```diff
@@ -4262,10 +4262,10 @@ bool CVerifyDB::VerifyDB(const CChainParams& chainparams, CCoinsView *coinsview,
     for (pindex = ::ChainActive().Tip(); pindex && pindex->pprev; pindex = pindex->pprev) {
         boost::this_thread::interruption_point();
         const int percentageDone = std::max(1, std::min(99, (int)(((double)(::ChainActive().Height() - pindex->nHeight)) / (double)nCheckDepth * (nCheckLevel >= 4 ? 50 : 100))));
-        if (reportDone < percentageDone/10) {
+        if (reportDone < percentageDone/1) {
```

Comment:
Maybe extract this as a constant that is set to `10` when nCheckDepth is less than 1000 blocks and to `1` otherwise?

Reply:
I agree with @MarcoFalke.

Reply:
@MarcoFalke  @laanwj   Done. Thank you.